.github/workflows/docker_builds.yml

```
@@ -15,22 +15,27 @@ jobs:
         python_version: [3.6, 3.7, 3.8]
         pytorch_version: [1.1, 1.2, 1.3, 1.4, 1.5]
     steps:
-    - name: Extract Current Tag
-      if: contains(github.ref, 'refs/tags/')
-      id: get_version
-      run: echo ::set-output name=VERSION::${GITHUB_REF/refs\/tags\//}
+    - name: Extract branch name
+      if: contains(github.ref, 'refs/tags/') != true
+      shell: bash
+      run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
+      id: extract_tag
+    - name: Extract Tag name
+      if: contains(github.ref, 'refs/tags')
+      shell: bash
+      run: echo "##[set-output name=tag;]$(echo ${GITHUB_REF#refs/tags/})"
     - uses: actions/checkout@v2
     - name: Publish Releases to Docker
       # only on releases
       uses: elgohr/Publish-Docker-Github-Action@2.14
-      if: contains(github.ref, 'refs/tags/') && !contains(${{ steps.get_version.outputs.VERSION }}, 'rc') %% !contains(${{ steps.get_version.outputs.VERSION }}, 'dev')
+      if: contains(github.ref, 'refs/tags/') && contains(${{ steps.extract_tag.outputs.tag }}, 'rc') != true && contains(${{ steps.extract_tag.outputs.tag }}, 'dev') != true
```

```suggestion
      if: contains(github.ref, 'refs/tags/') && !contains(${{ steps.extract_tag.outputs.tag }}, 'rc') && !contains(${{ steps.extract_tag.outputs.tag }}, 'dev')
```
not sure, need to check it...